rmf_fleet_adapter_python/examples/traffic_light.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from rclpy.node import Node
+
+import rmf_adapter as adpt
+import rmf_adapter.vehicletraits as traits
+import rmf_adapter.geometry as geometry
+import rmf_adapter.graph as graph
+import rmf_adapter.plan as plan
+import rmf_adapter.nodes as nodes
+
+import rmf_adapter.easy_traffic_light as traffic_light
+
+import time
+from threading import Thread
+
+
+def init_graph():
+    bot1_path = [1, 2, 3, 4, 5]
+    bot2_path = [0, 3, 4, 6]
+
+    test_graph_vis = \
+        """
+                         6 (bot2 end)
+                         |
+    (bot1 start)         |
+    1------2------3------4-----5 (bot1 end)
+                  |
+                  |
+                  0 (bot2 start)
+    """
+    print(test_graph_vis)
+    print(f"dummybot1 path {bot1_path}")
+    print(f"dummybot2 path {bot2_path}")
+    print("=================================================\n")
+    return bot1_path, bot2_path
+
+
+class MockTrafficLightHandle:
+    def __init__(self, name):
+        print(f" [{name}] Start Mock EZ traffic light")
+        self.name = name
+        self.handler = None
+        self.path_checkpoints = []
+        self.coors = [[3, 0, 0], [1, 1, 0], [2, 1, 0],
```

Comment:
I'd suggest naming this something like `positions` or `coordinates` instead to avoid unpaid advertisement of a beer company.